tests/test_vprec_backend/generic_test.sh

```diff
@@ -61,59 +67,53 @@ print_sep() {
 }
 
 compute_op() {
-    rm mpfr.txt
-    rm vprec.txt
     while read a b; do
 	./compute_mpfr_rounding.py $a $b $1 $2 >> mpfr.txt
 	./compute_vprec_rounding $a $b $1 >> vprec.txt
-    done < input.txt
+	done < input.txt
```

Comment:
I suppose this is still working, but I would say the previous indentation was the correct one.

Reply:
done